src/vm/eventpipe.h

```diff
@@ -29,6 +29,70 @@ typedef void (*EventPipeCallback)(
     void *FilterData,
     void *CallbackContext);
 
+struct EventData
+{
+public:
+    unsigned long ptr;
+    unsigned int size;
+    unsigned int reserved;
+};
+
+class EventPipeEventPayload
+{
+private:
+    BYTE *m_pData;
+    EventData **m_pBlobs;
+    unsigned int m_blobCount;
+    unsigned int m_size;
+    bool m_performedAllocation;
+
+public:
+    // Build this payload with a flat buffer inside
+    EventPipeEventPayload(byte *pData, unsigned int length);
```

Comment:
Addressed in https://github.com/dotnet/coreclr/pull/13347/commits/851943c5d5dc653dc1625a376c26bc2d6cf6a587